### ISSUE TITLE
Run 5 U1404

### ISSUE THREAD

**Issue by jenkinsSAGrid** [SouthAfricaDigitalScience/FMD-codes-from-T.-Neff-#5]
This was created via the API at the start of build 5

**Comment by brucellino** [comment]
failed in `deploy` with missing target veryclean : 

```
[0.1] $ /bin/bash -xe /tmp/hudson2724246193946400962.sh
+ cd CODE-RADE
+ ./deploy.sh

All tests have passed, will now build into /apprepo/generic/u1404/x86_64/FMD/0.1
make: *** No rule to make target `veryclean'.  Stop.
```

Since the `veryclean` target does exist, this is probably a workdir issue.

**Comment by brucellino** [comment]
Fixed in 2c1835a
